docs/get-started/pretrained.md

```diff
@@ -0,0 +1,48 @@
+(pretrained-models)=
+# {octicon}`rocket` Jina Embeddings
+
+Starting with Finetuner 0.7.9,
+we introduce a suite of pre-trained text embedding models licensed under Apache 2.0.
+The model have a range of use cases, including information retrieval, semantic textual similarity, text reranking, and more.
+The suite includes the following:
+
+- `jina-embedding-s-en-v1` **[Huggingface](jinaai/jina-embedding-s-en-v1)**: With a compact size of just 35 million parameters, the model enables lightning-fast inference while still delivering impressive performance.
```

Comment:
```suggestion
- `jina-embedding-s-en-v1` **[Huggingface](jinaai/jina-embedding-s-en-v1)**: This is a compact model with just 35 million parameters, that performs lightning-fast inference while delivering state-of-the-art performance.
```

Reply:
i'll apply the rest but not SOTA, because it;s not the top one